qsiprep/workflows/dwi/merge.py

```diff
@@ -311,7 +311,7 @@ def get_buffernode():
                 name='denoiser')
         else:
             denoiser = pe.Node(
-                Patch2Self(residuals=True, patch_radius=dwi_denoise_window),
+                Patch2Self(patch_radius=dwi_denoise_window),
```

Comment:
Hi @mattcieslak -- 1 thing that I wanted to ask is the default for `Patch2Self` when calling, I think its currently at 5 window size. The default should be set to `dwi_denoise_window=1`. The default patch radius needed for Patch2Self is 0 -- therefore the window size would be `2*0+1=1`, assuming an isotropic window.

Reply:
Turns out we're ignoring patch radius if window is set to "auto" (https://github.com/PennBBL/qsiprep/blob/master/qsiprep/interfaces/dipy.py#L138) and "auto" is the default (https://github.com/PennBBL/qsiprep/blob/master/qsiprep/cli/run.py#L192). Does this logic still work?

Reply:
Yes! Sorry! Din't see this 👍🏽 Looks great!